adyenofficial.php

```diff
@@ -92,6 +92,30 @@ class AdyenOfficial extends PaymentModule
      */
     private $adyenPaymentResponseModel;
 
+    /**
+     * Links to show on the Adyen payment method header
+     *
+     * @var array
+     */
+    private static $links = array(
+        array(
+            "label" => "Docs",
+            "url" => "https://docs.adyen.com/plugins/prestashop"
+        ),
+        array(
+            "label" => "Support",
+            "url" => "https://support.adyen.com/hc/en-us/requests/new?ticket_form_id=78764"
+        ),
+        array(
+            "label" => "GitHub",
+            "url" => "https://github.com/Adyen/adyen-prestashop/releases"
+        ),
+        array(
+            "label" => "PrestaShop Marketplace",
+            "url" => "https://addons.prestashop.com"
```

Comment:
Do we really have a marketplace link for Adyen? I cannot find it on addons.prestashop.com so we can update it here

Reply:
In order to find your module, please make sure that your location (at the bottom of the page) is set on the countries of France or Spain.
I updated the link, thanks for noticing!